.github/workflows/functional.yaml

```diff
@@ -3,6 +3,10 @@ name: functional
 on:
   workflow_call:
 
+concurrency:
```

Comment:
The functional yaml is integrated into control.yml therefore I don't think this is necessary here as it should be controlled by control.yml. See the on workflow_call.